core/eolearn/tests/test_core_tasks.py

```diff
@@ -406,8 +406,9 @@ def test_explode_bands(
 def test_extract_bands(patch: EOPatch) -> None:
     bands = [2, 4, 6]
     patch = ExtractBandsTask((FeatureType.DATA, "bands"), (FeatureType.DATA, "MOVED_BANDS"), bands)(patch)
+
     assert_array_equal(patch.data["MOVED_BANDS"], patch.data["bands"][..., bands])
-    assert id(patch.data["MOVED_BANDS"]) != id(patch.data["bands"])
+    assert all([(id(patch.data["MOVED_BANDS"][band]) != id(patch.data["bands"][band]) for band in bands)])
```

Comment:
I see you're very enthusiastic about `id` tests, but you're underestimating numpy.
Numpy can also return so called `views`. Look at the following snippet from the console:

```python
In [1]: import numpy as np

In [2]: x = np.ones((10, 10))

In [3]: y = x[3:5, 3:5]

In [4]: z = x[3:5, 3:5]

In [5]: id(y) == id(z)
Out[5]: False

In [6]: z[0, 0] = 0

In [7]: y
Out[7]: 
array([[0., 1.],
       [1., 1.]])
```